src/ThrowsAnnotationReader.php

```diff
@@ -93,12 +92,7 @@ public function readByReflection($reflection, Scope $scope): array
 	 */
 	private function parse($reflection, string $sourceFile, ?string $namespace = null): array
 	{
-		try {
-			$docBlock = $this->getDocblock($reflection);
-		} catch (ReflectionException $exception) {
```

Comment:
@marcospassos In which case can this happen?

Reply:
[Here](https://github.com/pepakriz/phpstan-exception-rules/blob/master/src/ThrowsAnnotationReader.php#L134) and [here](https://github.com/pepakriz/phpstan-exception-rules/blob/master/src/ThrowsAnnotationReader.php#L140).

In theory, PHPStan is free to create synthetic annotations, such as for magic methods, right?

Reply:
I know where it can occur, but I don't know the concrete case.

Reply:
Not sure if those cases exist currently, but it may exist.

Reply:
So instead of this, I'd like to convert this exception into `ShouldNotHappenException` inside `ThrowsAnnotationReader::getDocblock()` method. ok?

Reply:
Sure, we can revert it if we find a real case.